src/labels/CustomLabels.labels

```diff
@@ -7327,6 +7519,422 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of Prebuilt NPSP Report checklist item in Get Started for End Users</shortDescription>
+        <value>Reports Workbook</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemPrebuiltNPSPReportTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Prebuilt NPSP Report checklist item in Get Started for End Users</shortDescription>
+        <value>Run a Prebuilt NPSP Report</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of Standard Report Type checklist item in Get Started for End Users</shortDescription>
+        <value>The Lightning Report Builder gives you the ability to create your own custom reports using standard report types. The report type you choose determines the objects and fields available in your report.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of Standard Report Type checklist item in Get Started for End Users</shortDescription>
+        <value>Learn more about Standard Report Types in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypePriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Button label of Standard Report Type checklist item in Get Started for End Users</shortDescription>
+        <value>Create Reports</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemStandardReportTypeTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Title of Standard Report Type checklist item in Get Started for End Users</shortDescription>
+        <value>Create a Report from a Standard Report Type</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemViewDashboardDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of View Dashboard checklist item in Get Started for End Users</shortDescription>
+        <value>Dashboards pull data from one of more reports and display it side by side in the same window. NPSP includes three dashboards that you can use as is or customize.</value>
```

Comment:
Please fix the typo. The word "of" in the first sentence should be "or" so that the sentence reads: Dashboards pull data from one or more reports and display it side by side in the same window.